govdelivery/xml_response_parsers.py

```diff
@@ -14,36 +15,42 @@ def topic_xml_as_dict(unparsed_xml, only_listed=False):
     return code_map
 
 
+def listed_topic_xml_as_dict(unparsed_xml):
+    return topic_xml_as_dict(unparsed_xml, only_listed=True)
+
+
 def subscriber_topics_as_list(unparsed_xml):
     root = ET.fromstring(unparsed_xml)
     codes = []
+
     for topic in root.findall('topic'):
         codes.append(topic.find('to-param').text)
 
     return codes
 
 
-def visible_category_xml_as_dict(unparsed_xml):
-    return category_xml_as_dict(unparsed_xml, only_listed=True)
-
+# This parser is currently unused.
+# TODO: Write corresponding get_subscriber_answers client method.
```

Comment:
If this is unused, is there a reason to leave it in?

Reply:
Well, presumably it would be useful if we had a `get_subscriber_answers` method in the API client, and presumably we eventually want to build out methods to cover all of the features that the GovDelivery API offers.